signalfx-tracing/signalfx-nodejs-tracing/hapi_pg/README.md

```diff
@@ -0,0 +1,102 @@
+
+
+
```

Comment:
nit: do you mind removing extraneous whitespace?  Also I think a single blank line between segments in js files is an accepted convention (used in both google and airbnb guides).